.vsts-ci.yml

```diff
@@ -206,6 +206,14 @@ jobs:
             # Do not publish zips and tarballs. The linux-x64 binaries are
             # already published by Build_LinuxPortable_Release_x64
             _AdditionalBuildParameters: '/p:PublishBinariesAndBadge=false'
+          Build_Internal_Package_X64:
+            _BuildConfig: Release
+            _DockerParameter: ''
+            _LinuxPortable: ''
+            _RuntimeIdentifier: ''
+            _BuildArchitecture: 'x64'
+            _DropSuffix: ''
+            _AdditionalBuildParameters: --projects $(Build.SourcesDirectory)/src/Microsoft.Dotnet.Sdk.Internal/Microsoft.Dotnet.Sdk.Internal.csproj
```

Comment:
@tmat is there a general way to put a condition around a project (like the way that ExcludeFromSourceBuild works) in arcade? It looks like "_SuppressAllTargets" is private and only wired to ExcludeFromSourceBuild + DotnetBuildFromSource. I would like to consider putting a conditional on this poroject that is tied to one leg rather than adding a whole extra leg to the build.

Reply:
Would setting IsPackable conditionally work? Or is there another boolean in the project that means do not publish?

Reply:
Took a stab at how the idea of making IsPackable work conditionally. https://github.com/dotnet/core-sdk/pull/1832